Core/Loyc.Syntax/LES/LesLexer.cs

```diff
@@ -460,28 +462,72 @@ public static object ParseNumberCore(UString source, bool isNegative, int number
 			}
 		}
 
+		static object ParseBigIntegerValue(UString source, bool isNegative, int numberBase, ref string error)
+		{
+			BigInteger bigIntResult;
+			bool overflow = !ParseHelpers.TryParseUInt(ref source, out bigIntResult, numberBase, ParseNumberFlag.SkipUnderscores);
+			if (!source.IsEmpty) {
```

Comment:
Okay, a syntax error is not impossible in `ParseIntegerValue` since it is called by `ParseNumberCore` which is public - though it's not called by any of my other code and maybe it shouldn't be public. So, this code is OK since `ParseNumberCore` is public.